data/presets/presets/attraction/summer_toboggan.json

```diff
@@ -0,0 +1,18 @@
+{
+    "fields": [
+        "name",
+        "operator",
+        "opening_hours"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "terms": [
+        "alpine slide",
+        "mountain coaster"
+    ],
+    "tags": {
+        "attraction": "summer_toboggan"
```

Comment:
Reading on that wiki page ... does it also need `leisure=track` here?

Reply:
I'm not sure about that. The wiki indeed suggests it but a quick [overpass turbo query](http://overpass-turbo.eu/s/Dd0) shows that 460 of 715 ways do not have the leisure key present at all (and this doesn't neccesarily mean, that the remaining 255 objects leisure tag is `leisure=track`).

Reply:
ok, yeah seems fine to me to leave it off.  [`attraction=water_slide`](https://wiki.openstreetmap.org/wiki/Tag:attraction%3Dwater_slide) seems like almost the same thing, and doesn't say anything about adding extra tags to it.